src/viam/examples/modules/complex/base/impl.hpp

```diff
@@ -0,0 +1,51 @@
+/// @file complex/base/impl.hpp
+///
+/// @brief Implements a `Base` component `MyBase`.
+///
+/// `MyBase` inherits from the `Base` class defined in the viam C++ SDK and
+/// implements some of the relevant methods along with `reconfigure`. It also
+/// specifies a static `validate` method that checks config validity.
```

Comment:
I chose to explain the files and some classes with these doc comments. Unclear if these will actually appear anywhere in cpp.viam.dev, though.

Reply:
You can test to find out! https://github.com/viamrobotics/viam-cpp-sdk#building-documentation-locally-for-testing

Reply:
They all appear in cpp.viam.dev! Thanks for linking me to those instructions.